podcasts/NotificationsViewController.swift

```diff
@@ -30,45 +71,73 @@ class NotificationsViewController: PCViewController, UITableViewDataSource, UITa
+        case .featuresAndOffers:
+            return rows[section].count
+        case .recommendationsAndReminders:
+            return rows[section].count
+        }
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        if indexPath.row == 0 {
+        guard let sectionType = Section(rawValue: indexPath.section) else {
+            return UITableViewCell()
+        }
+        let row = rows[sectionType.rawValue][indexPath.row]
+        switch row {
+        case .newEpisodes:
             let cell = tableView.dequeueReusableCell(withIdentifier: switchCellId, for: indexPath) as! SwitchCell
-            cell.cellLabel.text = L10n.newEpisodes.localizedCapitalized
+            cell.cellLabel.text = row.description
             cell.cellSwitch.isOn = NotificationsHelper.shared.pushEnabled()
 
             cell.cellSwitch.removeTarget(self, action: nil, for: UIControl.Event.valueChanged)
             cell.cellSwitch.addTarget(self, action: #selector(pushToggled(_:)), for: UIControl.Event.valueChanged)
 
             return cell
-        } else if indexPath.row == 1 {
+        case .podcastsChosen:
             let cell = tableView.dequeueReusableCell(withIdentifier: disclosureCellId, for: indexPath) as! DisclosureCell
             let podcastsSelected = DataManager.sharedManager.pushEnabledPodcastsCount()
             let chosenPodcasts = podcastsSelected == 1 ? L10n.chosenPodcastsSingular : L10n.chosenPodcastsPluralFormat(podcastsSelected.localized())
             cell.cellLabel.text = (podcastsSelected == 0) ? L10n.filterChoosePodcasts : chosenPodcasts
             cell.cellSecondaryLabel.text = nil
 
             return cell
-        }
-
-        let cell = tableView.dequeueReusableCell(withIdentifier: disclosureCellId, for: indexPath) as! DisclosureCell
-        cell.cellLabel.text = L10n.appBadge
-        let badgeChoice = Settings.appBadge
-
-        switch badgeChoice {
-        case .totalUnplayed:
-            cell.cellSecondaryLabel.text = L10n.statusUnplayed
-        case .filterCount:
-            cell.cellSecondaryLabel.text = L10n.settingsNotificationsFilterCount
-        case .newSinceLastOpened:
-            cell.cellSecondaryLabel.text = L10n.newEpisodes
+        case .appBadges:
+            let cell = tableView.dequeueReusableCell(withIdentifier: disclosureCellId, for: indexPath) as! DisclosureCell
+            cell.cellLabel.text = row.description
+            let badgeChoice = Settings.appBadge
+
+            switch badgeChoice {
+            case .totalUnplayed:
+                cell.cellSecondaryLabel.text = L10n.statusUnplayed
+            case .filterCount:
+                cell.cellSecondaryLabel.text = L10n.settingsNotificationsFilterCount
+            case .newSinceLastOpened:
+                cell.cellSecondaryLabel.text = L10n.newEpisodes
+            default:
+                cell.cellSecondaryLabel.text = L10n.off
+            }
```

Comment:
What if we extend `AppBadge` outside its module to have an enum computed property `var description: String { ... }`? This will reduced by one line using it